Cargo.toml

```diff
@@ -16,6 +16,8 @@ readme = "README.md"
 futures-core = { version = "0.3.1", optional = true }
 futures-sink = { version = "0.3.1", optional = true }
 futures-01 = { version = "0.1", optional = true, package = "futures" }
+tokio = { version = "0.2", optional = true }
+#tokio-01 = { version = "0.1", optional = true, package = "tokio" }
```

Comment:
Could you please remove two commented references to tokio-01?

Reply:
Yeah I thought about removing it first but then I left them there... Let me do that right now